frontend/src/libs/components/input/input.tsx

```diff
@@ -70,9 +71,10 @@ const Input = <T extends FieldValues>({
         disabled={isDisabled}
         inputMode={inputMode}
       />
-      <span className={styles['error-message']}>
-        {hasError && (error as string)}
-      </span>
+      <ErrorMessage
+        error={error as string}
+        className={styles['error-message']}
```

Comment:
If this custom className brings only a `min-height` for container - won't it be better to put it inside `ErrorMessage` styles?